content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,168 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Policy reports provide information about policy execution and violations. Use '--policy_report' with the apply command to generate policy report.
+
+Policy report can also be generated for a live cluster. While generating policy report for a live cluster the '-r' flag is assuming a resource by specific name which is assumed to be globally unique. And it doesn't support naming the resource type (ex., Pod/foo when the cluster contains resources of different types with the same name). To generate policy report for a live cluster use '--cluster' with '--policy_report'.
+
+Example:
+```sh
+kyverno apply policy.yaml --cluster --policy_report
+```
+Above example applies a `policy.yaml` to all resources in the cluster.
+
+Input combination:
+
+Below are the combination of inputs that can be used for generating the policy report from Kyverno CLI.
+
+| Policy        | Resource         | Cluster   | Namespace      | Interpretation                                                                           |
+| ---- |:-------------:| :---------------:| :--------:| :-------------:| :----------------------------------------------------------------------------------------| 
+| policy.yaml   | -r resource.yaml | false     |                | Apply policy from `policy.yaml` to the resources specified in `resource.yaml`                                         |
+| policy.yaml   | -r resourceName  | true      |                | Apply policy from `policy.yaml` to the resource with a given name in the cluster                                        |
+| policy.yaml   |                  | true      |                | Apply policy from policy.yaml to all the resources in the cluster                                   |
```

Comment:
This line's Interpretation doesn't seem to be left aligning like the table indicates. Not sure what the deal is here...